From e3437608db7c7f13938e4ebbb7639b41bef58124 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Sun, 5 Apr 2015 19:28:56 +0200
Subject: Fix build for Haiku.

* We do have glext.h and it conflicts with the SDL replacement.

diff --git a/src/makefile b/src/makefile
index 3b20757..248715f 100644
--- a/src/makefile
+++ b/src/makefile
@@ -79,7 +79,7 @@ endif
 # Haiku
 ifeq ($(PLATFORM),haiku)
 LFLAGS += `sdl-config --libs` -lGL -lSDL_mixer
-CFLAGS += `sdl-config --cflags`
+CFLAGS += `sdl-config --cflags` -DNO_SDL_GLEXT
 endif
 
 # Pandora
-- 
2.2.2

